### PR TITLE
 luci-mod-admin-full: dnsmasq clarify non-wildcard mode

### PR DESCRIPTION
**Pull request by ldir-EDB0**
'non-wildcard' interfaces enables dnsmasq's '--bind-dynamic' mode.
This binds to interfaces rather than wildcard addresses *and* keeps track
of interface comings/goings via a unique Linux api.

Quoting dnsmasq's author "bind-dynamic (bind individual addresses,
keep up with changes in interface config) ... On linux, there's actually
no sane reason not to use --bind-dynamic, and it's only not the default
for historical reasons."

Removed the dependency of 'nonwildcard' enabling access to
'listen/exclude' interfaces.  This was bogus anyway, dnsmasq has no
requirement for 'bind dynamic' to enable include/exclude interfaces.
In fact the dnsmasq init script takes notice of include/exclude
interfaces irrespective of the 'nonwildcard' parameter.

Signed-off-by: Kevin Darbyshire-Bryant <kevin@darbyshire-bryant.me.uk>

### CODE REVIEW

**Comment by ldir-EDB0** [comment]
Another loitering PR - will someone *PLEASE* look at it.  I happen to think it's sensible but then maybe I'm not.

**Comment by karlp** [comment]
I suggest rewording the help text for "non-wildcard" too, to be more clear as the dnsmasq author's notes say

**Comment by ldir-EDB0** [comment]
@hnyman This has been hanging around for a while, could it be merged please?
